server/game/gamesteps/actionwindow.js

```diff
@@ -22,9 +22,9 @@ class ActionWindow extends PlayerOrderPrompt {
 
     activePrompt() {
         return {
-            menuTitle: 'Any actions or reactions?',
+            menuTitle: 'Initiate an action, or pass',
             buttons: [
-                { text: 'Done' }
+                { text: 'Pass' }
```

Comment:
I like that it explicitly says Pass now. Changing `GameFlowWrapper.skipActionWindow` should hit 99% of the failures, there may be a couple that need to be updated by hand.